.github/workflows/aws-deploy.yaml

```diff
@@ -12,70 +12,26 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
-      - uses: pre-commit/action@v2.0.0
-  sceptre-sandbox:
-    if: github.event_name == 'push'
+      - uses: pre-commit/action@v2.0.2
+
+  sceptre-deploy:
+    name: Deploy CloudFormation templates using sceptre
     runs-on: ubuntu-latest
     needs: pre-commit
+    if: github.ref == 'refs/heads/prod'
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
-        with:
-          python-version: 3.8
-      - name: Cache python modules
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install sceptre sceptre-ssm-resolver
```

Comment:
why you remove this?  you need to install sceptre to us it to deploy cfn templates.

Reply:
See above comment about `sceptre-action`.